src/main/java/org/kainos/ea/api/ResponsibilityService.java

```diff
@@ -32,4 +34,16 @@ public List<Responsibility> getResponsibilitiesForJob(int jobId) throws FailedTo
             throw new FailedToGetResponsibilitiesException();
         }
     }
+    // New method to check if a job exists
+    public boolean doesJobExist(int jobId) throws SQLException {
+        Connection connection = null;
+        try {
+            connection = databaseConnector.getConnection();
+            return responsibilityDao.doesJobExist(connection, jobId);
+        } finally {
```

Comment:
you don't need to do this, we can re-use the connection

Reply:
Done.